samcli/lib/utils/colors.py

```diff
@@ -0,0 +1,38 @@
+
+import click
+
+
+class Colored(object):
```

Comment:
Doesn't this tie us deeper into click? I know we only wanted click related items to be in the commands only.